delivery_carrier_label_postlogistics/postlogistics/web_service.py

```diff
@@ -44,12 +61,29 @@ def __init__(self, company):
         self.init_connection(company)
 
     def init_connection(self, company):
-        t = HttpAuthenticated(
-            username=company.postlogistics_username,
-            password=company.postlogistics_password)
-        self.client = Client(
-            company.postlogistics_wsdl_url,
-            transport=t)
+        if company.postlogistics_test_mode:
+            # We must change location for namespace as suds will take the wrong
+            # one and find nothing
+            # In order to have the test_mode working, it is necessary to patch
+            # suds with https://fedorahosted.org/suds/attachment/ticket/239/suds_recursion.patch # noqa
```

Comment:
This module is already based on suds jurko fork. This is not the official patch in there dev branch, but it works. suds-jurko has another patch which didn't worked well in this case.

The patch is required mostly to develop the module and requires to have access to a test account which must be asked so it should touch few users.

Ok I'll add a note in __openerp__.py (no readme.rst in 7.0) and add a copy of the patch to apply in a patch folder.